modules/ROOT/pages/configure-service-mesh.adoc

```diff
@@ -35,108 +36,144 @@ After the new adapter is ready, the old pod is terminated and all requests are r
-For more information, see https://docs.mulesoft.com/api-manager/2.x/create-instance-task
-. If your applications were already running in the namespace, restart your application:
-`$ kubectl delete pod -n marketing -l app=campaigns`
+. Obtain the API ID of the API that you want to bind with the service.
+. If your applications were already running in the namespace, restart your application to inject the sidecar in the POD:
++
+`$ kubectl -n <namespace> patch deploy <deployname> --type=json -p='[{"op": "replace", "path": "/spec/template/metadata/labels/service-mesh.mulesoft.com","value":"enable"}]`
++
+In the above command, replace the namespace and the deploy name with your specific details.
 +
 . Verify that the application Kubernetes service port is named following the requirements requested by Istio: https://istio.io/docs/setup/additional-setup/requirements/ .
 . From the Service Mesh installer download location in the Command window, type the following command to bind the adapter with an application:
 +
 [source,text,linenums]
 ----
-./service-mesh binding create \
---name=<replace_with_servicename>-anypoint-security \
---namespace=marketing \same as adapter creation
---adapter=anypoint-service-mesh-adapter \same as adapter creation
---apiId=<replace_with_apiId> \
---serviceName=<replace_with_servicename>
+./service-mesh binding create 
+--name=<servicename>
+--namespace=<namespace name>
+--adapter=<adapter name>
+--apiId=<application ID> 
+--serviceName=<servicename>
 ----
 +
-Where:
-+
 [%header%autowidth.spread]
 |===
 | Parameter Name | Description | Required/Optional | Default Value
-| name | The name that identifies the binding that you want to create, for example, “test-binding-anypoint-security”. For the Beta exercise, replace the text within the angular braces. | Required | N/A
+| name | The name that identifies the binding that you want to create, for example, service-binding-anypoint-security”. 
+ | Required | N/A
 | namespace | The Kubernetes namespace in which you want to create the binding, for example, “mulesoft-apps”. Make sure that you use the same namespace as the adapter you provisioned in the previous procedure. | Required | N/A
 | adapter | Specify the name of the adapter that you provisioned in the previous procedure.  | Required | N/A 
-| apiId | Specify the API Id of the API that you want to associate with this adapter.   | Required | N/A.
+| apiId | Specify the API ID of the API that you want to associate with this adapter. | Required | N/A.
  Use the API ID you obtained as part of installation prerequisites as explained in this document.
-| serviceName | The name of the service that will be binded to the API to be protected and managed by Anypoint Platform. | Required | N/A.
-Use the client Id and client secret you obtained as part of installation prerequisites as explained in this document.
+| serviceName | The name of the service to be binded to the API, which will be managed by Anypoint Platform. | Required | N/A.
```

Comment:
The name of the service to be bound to the API, which will be managed by the Anypoint Platform.

Reply:
Done!